### PR TITLE
ESC-1136 - update content and add feature flag for XI EORI removal

### PR DESCRIPTION
**Pull request by masahmad**
- update content and add feature flag for XI EORI removal

### CODE REVIEW

File: app/uk/gov/hmrc/eusubsidycompliancefrontend/config/AppConfig.scala [hunk 1]
```diff
@@ -31,6 +31,9 @@ class AppConfig @Inject() (config: Configuration, contactFrontendConfig: Contact
   val euroOnlyEnabled: Boolean =
     config.getOptional[Boolean]("features.euro-only-enabled").getOrElse(false)
 
+  val xiEoriAdding: Boolean =
```
Comment (habeeb01) [inline app/uk/gov/hmrc/eusubsidycompliancefrontend/config/AppConfig.scala:34]: sorry to be pedantic but can we rename the flag to `xi-eori-enabled` and `xiEoriEnabled` (i just think it reads a bit better). And we typically do `getOrElse(false)`. Lets stay consistent.

File: app/uk/gov/hmrc/eusubsidycompliancefrontend/views/BusinessEntityEoriPage.scala.html [hunk 2]
```diff
@@ -52,6 +54,8 @@
       )
   }
 
+@if(appConfig.xiEoriAdding) {
```
Comment (habeeb01) [inline app/uk/gov/hmrc/eusubsidycompliancefrontend/views/BusinessEntityEoriPage.scala.html:57]: this is the wrong way around. We want the old behaviour by default and only see the new functionality when the flag is true

File: app/uk/gov/hmrc/eusubsidycompliancefrontend/views/BusinessEntityEoriPage.scala.html [hunk 3]
```diff
@@ -68,6 +72,34 @@
...
+} else {
+
+    @H1(messages("businessEntityEoriNew.title"), id = Some("businessEntityEoriTitleId"))
+    @P(messages("businessEntityEori.p1"), id = Some("businessEntityEoriP1Id"))
+    @P(messages("businessEntityEori.p2"), id = Some("businessEntityEoriP2Id"))
+    @P(messages("businessEntityEori.p3"), id = Some("businessEntityEoriP3Id"))
+
+
+    @govukInput(Input(
+    value = form.data.get("businessEntityEori"),
```
Comment (habeeb01) [inline app/uk/gov/hmrc/eusubsidycompliancefrontend/views/BusinessEntityEoriPage.scala.html:86]: try to indent/format these lines so its easier to read

File: app/uk/gov/hmrc/eusubsidycompliancefrontend/config/AppConfig.scala (hunk 1 above)
Comment (masahmad) [reply app/uk/gov/hmrc/eusubsidycompliancefrontend/config/AppConfig.scala:34]: its currently enabled so default is enabled in future we plan to disable
so would this read better
xiEoriAddingDisabled
and... features.xi-eori-adding-disabled

Comment (masahmad) [reply app/uk/gov/hmrc/eusubsidycompliancefrontend/config/AppConfig.scala]: updated

File: app/uk/gov/hmrc/eusubsidycompliancefrontend/views/BusinessEntityEoriPage.scala.html (hunk 2 above)
Comment (masahmad) [reply app/uk/gov/hmrc/eusubsidycompliancefrontend/views/BusinessEntityEoriPage.scala.html]: updated

File: app/uk/gov/hmrc/eusubsidycompliancefrontend/views/BusinessEntityEoriPage.scala.html (hunk 3 above)
Comment (masahmad) [reply app/uk/gov/hmrc/eusubsidycompliancefrontend/views/BusinessEntityEoriPage.scala.html]: updated

**Review by habeeb01 — APPROVED** [review]

**Pull request merged**